manifests/init.pp

```diff
@@ -57,6 +57,22 @@
   Array[String]  $users_deny     = [],
   Boolean        $manage_users_allow = false,
   Boolean        $manage_users_deny  = false,
+  Boolean        $manage_crontab     = false,
+  String         $crontab_shell      = '/bin/bash',
+  String         $crontab_path       = '/sbin:/bin:/usr/sbin:/usr/bin',
+  String         $crontab_mailto     = 'root',
+  Optional[String] $crontab_home     = undef,
+  Hash[
```

Comment:
Could you move this into a custom datatypes? Example: https://github.com/voxpupuli/puppet-zabbix/blob/master/types/databases.pp

Reply:
Converted to data type